### PR TITLE
Add CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by meain**
### What does this PR change?

Initial version of CONTRIBUTING.md. @techmaharaj feel free to make and updates on the same branch.

### Does the PR depend on any other PRs or Issues? If yes, please list them.

-

### Checklist

I confirm, that I have...

- [ ] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`
- [ ] Ran [`helm-docs`](https://github.com/norwoodj/helm-docs) to update docs for chart (if values.yaml file was changed)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+The next step is to **create a Kind cluster**. To do that you can create a copy of [this configuration file](https://github.com/paralus/helm-charts/blob/main/docs/kind.config.yaml) and use that to create a cluster.
+
+```bash
+kind create cluster --config cluster.yaml
+```
+
+*You can optionally pass name via `--name <name>` to create a kind cluster with a specific name.*
+
+### Install paralus from helm chart source
```
Comment (techmaharaj) [inline CONTRIBUTING.md:42]:
```suggestion
### Install Paralus from helm chart source
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+You can install the helm chart from source using the following command once you have cloned the repo.
+
+``` bash
+# helm install -n <namesapce> <installation_name> ./charts/ztka --values <custom_values_file> --create-namespace
+helm install -n ztka ztka ./charts/ztka --values examples/values.dev-generic.yaml --create-namespace
+```
+
+You should now have `paralus` deployed. You can check out [blog](https://www.paralus.io/blog/kind-quickstart) for a detailed guide on how to set up the rest of the application.
```
Comment (techmaharaj) [inline CONTRIBUTING.md:51]:
```suggestion
You should now have `paralus` deployed. You can check out our blog post to [setup Paralus on Kind](https://www.paralus.io/blog/kind-quickstart) for a detailed guide on how to set up the rest of the application.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Run the following command to regenerate the docs. Make sure to run this before you submit a pull request.
+
+``` shell
+helm-docs --chart-search-root=charts
+```
+
+## Need Help?
+
+If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/relay/issues/new) in this repository describing your issue and we'll take it up from there.
```
Comment (techmaharaj) [inline CONTRIBUTING.md:67]: Please correct the issue link
```suggestion
If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/helm-charts/issues/new) in this repository describing your issue and we'll take it up from there.
```

**Review by techmaharaj — COMMENTED** [review]
LGTM, just a few minor changes.

**Comment by meain** [comment]
@niravparikh05 @techmaharaj If there is nothing else, can we merge it in after an approving review.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Run the following command to regenerate the docs. Make sure to run this before you submit a pull request.
+
+``` shell
+helm-docs --chart-search-root=charts
+```
+
+## Need Help?
+
+If you are interested to contribute to prompt but are stuck with any of the steps, feel free to reach out to us. Please [create an issue](https://github.com/paralus/helm-charts/issues/new) in this repository describing your issue and we'll take it up from there.
```
Comment (niravparikh05) [inline CONTRIBUTING.md:67]: I think you meant if you are interested to contribute to helm-charts ?

**Review by niravparikh05 — CHANGES_REQUESTED** [review]
correct the reference to helm-charts from prompt.

File: CONTRIBUTING.md (hunk 4 above)
Comment (meain) [reply CONTRIBUTING.md:67]: Updated

**Review by niravparikh05 — APPROVED** [review]
LGTM

**Pull request merged**